js/nav-category.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function () {
+  const navCategory = $('.nav-category');
+  if (navCategory.length > 0) {
+    const linkSearch = navCategory.find('.link-search');
+    if (linkSearch) {
```

Comment:
please add an additional event when user click on the icon again to close the search box. Please in CSS also remove the underline decoration on HOVER and when the searchbox is open the color must be like others inactive link

Reply:
updated, please kindly check on new commit